core/src/test/java/org/infinispan/distexec/DistributedExecutorFailoverTest.java

```diff
@@ -124,4 +160,25 @@ public int maxFailoverAttempts() {
          return maxFailoverCount;
       }
    }
+
+   static class SameNodeTaskFailoverPolicy implements DistributedTaskFailoverPolicy {
+      private final int maxFailoverCount;
+      private int failoverCount = 0;
+
+      public SameNodeTaskFailoverPolicy(int maxFailoverCount) {
+         super();
```

Comment:
No. It's the same with the original failover policy. So I think it should keep the same way to code.